src/asyncify/asyncify_impl.c

```diff
@@ -86,13 +86,42 @@ static void fiber_stack_free(struct fiber_stack fiber_stack) {
   free(fiber_stack.buffer);
 }
 
+#if defined USE_STACK_POOLING && USE_STACK_POOLING == 1
```

Comment:
It seems to be more idiomatic to use `#ifdef` for feature flags in C.
You could do the following in the makefile, similar to what I'm doing for the shadow stack:
```
ifeq ($(USE_STACK_POOLING),1)
  STACK_POOLING_FLAGS=-DUSE_STACK_POOLING -DSTACK_POOL_SIZE=$(STACK_POOL_SIZE)
else
  STACK_POOLING_FLAGS=
endif
```

Reply:
On second thought, we could fuse this into a single flag: `USE_STACK_POOLING=0` means on the demand allocation, and `USE_STACK_POOLING=N` for `N > 0` means stack pooling with max `N` stacks for now?

Reply:
I resisted that temptation when it came to the shadow stack flags... Having a boolean-sounding flag whose non-zero values are not equivalent seems unintuitive.

Reply:
Actually, how about using `STACK_POOL_SIZE` as the name of the unified flag? Having `0` mean "disabled" then sounds reasonable

Reply:
We can do that!